sources/.prod/on_deploy.sh

```diff
@@ -1 +1,3 @@
-#!/bin/bash
+#!/bin/sh
+
+exec python /app/manage.py migrate --noinput
```

Comment:
Is there a reason why the "standard" on_deploy doesn't work here? :thinking:

```suggestion
#!/bin/bash
set -e

./manage.py migrate --noinput
```